src/app/collection/shared/included/collection-learning-object-card/collection-learning-object-card.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '~_vars.scss';
+.container{
+    width: 20vw;
+    margin-bottom: 50px;
+    display: flex;
+}
+
+
+
+.header{
+    &.module{
+        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380642.jpg");
+    
+    }
+    &.micromodule{
+        background-image: url('../../../../../assets/images/adi-goldstein-EUsVwEOsblE-unsplash\ 2.png');
+    
+    }
+    &.nanomodule{
+        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380618.jpg");
+    
+    }
+    &.unit{
+        background-image: url("../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash\ 1.png");
+    
+    }
+    &.course{
+        background-image: url("../../../../../assets/images/pexels-scott-webb-430208.jpg");
+    
+    }
```

Comment:
I would still add some type of margin between the description and image

Reply:
I agree. 10px-15px should do the trick.

Reply:
i added that in a seperate file, but i can put that here instead